content/the-ember-times-issue-183.md

```diff
@@ -31,15 +34,11 @@ Try out uELS today by installing over [Open VSX](https://open-vsx.org/extension/
 
 ---
 
-## [2. Section title in sentence case 🐹](section-url)
-
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+## Around the web 🌐
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+* Using [empress-blog](https://github.com/empress/empress-blog)? If you're not familiar with it, empress-blog is the fully-functional, SEO friendly static site implementation of a blog system built on Ember. Check out [Michal Bryxí (@MichalBryxi)](https://github.com/MichalBryxi)'s bite-sized dev.to posts on [how to set up Mixpanel tracking events](https://dev.to/michalbryxi/mixpanel-empress-blog-5fd2) and [how to add a tag page to your main menu](https://dev.to/michalbryxi/tag-page-in-main-menu-for-empress-blog-31b9). Happy empress-blog-ing!
```

Comment:
Maybe start this block a bit different. I'm now kinda lost where it is going. Especially since the title of the part is also a bit ambiguous :)

Reply:
@MinThaMie Made some changes, what do you think about "Addon roundup"? Basically, they weren't long enough for a full writeup so kinda grouped these together. (Actually may add a couple more bullet points to there also.) Let me know if you want more changed!

Reply:
@MinThaMie sending out one day early so going to merge in, let me know what you think though, can make another PR with edits!